config/auto_battle_state_handler/速切模板-11号.sample.yml

```
@@ -35,16 +35,23 @@ handlers:
 
       - states: "[席德-能量]{0, 150}"
         sub_handlers:
-        - states: "[11号-特殊技可用]"
-          interrupt_states: "![11号-特殊技可用]"
+        - states: "[11号-终结技可用]"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)